en-us/server-setup.md

```diff
@@ -27,15 +27,14 @@ Minecraft 1.19.1 added a feature [that lets users report chat messages to Mojang
 
 #### How to protect your users
 
-- **All servers**: set `enforce-secure-profile` to `false` in _server.properties_
+- **All 1.19+ backend servers**: set `enforce-secure-profile` to `false` in _server.properties_
```

Comment:
could mention 1.18- too then that it doesn't need any changes

Reply:
Resolved.

Reply:
don't see the mention